samples/bluetooth/peripheral_uart/src/uart_async_adapter.c

```diff
@@ -0,0 +1,554 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *  @brief UART asynchronous API adapter implementation
+ */
+#include "uart_async_adapter.h"
+#include <drivers/uart.h>
+#include <sys/__assert.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(uart_async_adapter, LOG_LEVEL_ERR);
```

Comment:
In this file almost all logs have debug level so to print any log user need to modify this line.
This is a part of sample so maybe just:
```suggestion
LOG_MODULE_REGISTER(uart_async_adapter);
```
Then it should follow a general log level.
The other option is to make log level configurable for it by Kconfig

Reply:
OK